sanitizer/sanitizeAll.js

```diff
@@ -41,8 +41,8 @@ function checkParameters(req, sanitizers, cb) {
 
     // checks if there is a function that returns valid params
     if (typeof sanitizers[s].expected === 'function'){
-      /** func returns {array} ex: [{ name: 'text' }, { name: 'parsed_text' }] */
-      for (let t in sanitizers[s].expected()) {
+      /** func returns {array} ex: [{ name: 'text' }] */
+      for (var t in sanitizers[s].expected()) {
```

Comment:
if you are having to change `let` to `var` to make the travis tests work it's probably because `'use strict';` is not at the top of the file. Unfortunately Node.js 4 (but not Node.js 6) requires this in order to enable some ES6 features. Definitely use `let`